build.sbt

```diff
@@ -7,9 +7,9 @@ version := "0.1-SNAPSHOT"
 
 scalaVersion := "2.12.1"
 
-persistLauncher in Compile := true
+scalaJSUseMainModuleInitializer in Compile:= true
 
-persistLauncher in Test := false
+scalaJSUseMainModuleInitializer in Test := false
```

Comment:
This should only be
```scala
scalaJSUseMainModuleInitializer := true
```
instead of the two `scalaJSUseMainModuleInitializer` lines.